go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/giantswarm/kustomize-plugin-releasenotesurlannotationtransform
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.4.14
+	k8s.io/apimachinery v0.18.6
```

Comment:
I think we're targeting v1.18.5 for all of our repos.

Reply:
https://github.com/giantswarm/giantswarm/blob/232aef0faa24cd0f44d0125da8f2061b5e3e427c/content/docs/dev-and-releng/client-go/k8s-1-18/_index.md

Reply:
Do you know where this is documented?

Reply:
Docs are still in progress, see the link above though.

Reply:
thanks! Downgraded to 1.18.5